notebooks/1_intelligent_agents_fall_2021/index.md

```diff
@@ -0,0 +1,194 @@
+| Subject Tutoring       | Maximize scores, Improvement is students | Classroom, Desk, Chair, Board, Staff, Students |  Smart displays, Corrections | Eyes, Ears, Notebooks |
+
+# Properties of task environments
+
+#### Types of environment
+we can categorize an environment in many ways, you will find some of the most important ones listed below.
+
+<ul>
+  <li><b>Fully observable or partially observable</b> (Do the agent sensors give access to the complete state of the environment at each time?)</li>  
+    <ul> 
+        <li>We say an environment is fully observable if an agent has access to the complete state of the environment at each time.</li>
+        <li>If an agent doesn't know the complete state of the environment we say the environment is partially observable.</li>
+        <li>In the most extreme case when an agent knows nothing about the state of the environment we say the environment is unobservable.</li>
+        <li><b>examples</b>: In the vacuum world example our agent doesn't know if there is dirt in another room so this environment is partially observable for our agent. 
+    A simple chess agent has complete access to the chessboard's information at any given time so this environment is fully observable for our agent.</li>
+    </ul>
+    <br>    
+          
+  <li><b>Single agent or multiagent</b> (Are there more than one agent in the environment?)</li>
+    <ul>
+        <li>We say an environment is a multiagent environment if there is more than one agent operating in it otherwise we say the environment is sigle agent.</li>
+        <li>In some cases, we can model our environment both as a single agent and multiagent environment. For example, imagine an automatic taxi agent. Should this agent treat the other cars as objects or as another agent? It's  better to model our environment as a multiagent environment if the behavior of the other entities can be modeled as an agent seeking to maximize its performance measure which is somehow affected by our agent.</li>
+        <li>a multiagent environment could be competitive or cooperative or even a mix of both.</li>
+        <li><b>examples</b>: chess and automatic driving are multiagent environments. solving a crossword puzzle is a single agent environment.</li>
+    </ul>
+    <br>  
+
+  <li><b>Deterministic or stochastic</b> (Is the next state completely determined by the current state and the executed action?)</li>
+    <ul>
+        <li>We say an environment is deterministic if the next state is completely determined by the current state and the agent action otherwise, it is stochastic.</li>
+        <li><b>examples</b>: Chess is a deterministic environment. Automatic driving is stochastic because an agent can't predict everything(e.g traffic, accidents).</li>
+    </ul>  
+    <br>
+   
+  <li><b>Episodic or sequential</b> (Is the agent's experience divided into atomic "episodes“ where the choice of action in each episode depends only on the episode itself?)</li>
```

Comment:
double quotations are not in the same format, the first one is **"** while the other is **“**